format/Message.fbs

```diff
@@ -78,6 +214,30 @@ table DictionaryBatch {
   isDelta: bool = false;
 }
 
+table Xxhash64 {
+}
+
+// Types of digests accepted.
+union DigestType  {
+  Xxhash64 = 1
+  /// See documentation on BufferEncodingType for slot numbering and reservation
+  /// scheme.
+}
+
+/// Contains a digest of the immediately preceding message in the stream.
+/// The main use-case for digest is for interchange formats where there
+/// is no external means of ensuring data fidelity (e.g. a file streamed
+/// to a remote storage service).
+///
+/// Digest messages are optional.  The producer for a stream may decide
+/// to include them or not.
+table Digest {
+  digest_type : DigestType;
+
+  // Digest of preceding record batch.
```

Comment:
Is it just the record batch or the whole message?

Reply:
its the whole message.  reworded.